lib/FormBody.php

```diff
@@ -124,7 +124,11 @@ private function generateMultipartFileHeader(string $name, string $fileName, str
 
     private function generateMultipartFieldHeader(string $name, string $contentType): string {
         $header = "Content-Disposition: form-data; name=\"{$name}\"\r\n";
-        $header .= "Content-Type: {$contentType}\r\n\r\n";
+        if (!empty($contentType)) {
```

Comment:
Doesn't really matter here, because `0` isn't a valid content type, but please change it to `$contentType !== ""`, see https://3v4l.org/pF4sm.